# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/thunderbird/hotfix-cbindgen.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- thunderbird-102.0.3/gfx/webrender_bindings/webrender_ffi.h.orig	2022-08-24 22:48:16.059341841 +0200
+++ thunderbird-102.0.3/gfx/webrender_bindings/webrender_ffi.h	2022-08-24 22:49:57.574349988 +0200
@@ -73,8 +73,6 @@
 struct WrPipelineIdAndEpoch;
 using WrPipelineIdEpochs = nsTArray<WrPipelineIdAndEpoch>;
 
-const uint64_t ROOT_CLIP_CHAIN = ~0;
-
 }  // namespace wr
 }  // namespace mozilla
 
